### PR TITLE
Fixes #19560: Rename the AUTHORIZED_NETWORKS parameter used in the common system techniques.

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/19560

### CODE REVIEW

File: techniques/system/common/1.0/cf-serverd.st [hunk 1]
```diff
@@ -92,9 +92,9 @@ bundle common def
     policy_server::
       "acl" slist => {
         "127.0.0.0/8" , "::1", @{sys.ip_addresses},
-&if(AUTHORIZED_NETWORKS)&
+&if(ALLOWEDNETWORKS)&
```
Comment (ncharles) [inline techniques/system/common/1.0/cf-serverd.st:95]:
```suggestion
&if(ALLOWED_NETWORKS)&
```

**Review by ncharles — CHANGES_REQUESTED** [review]
it is not ALLOWEDNETWORKS but ALLOWED_NETWORKS

File: techniques/system/common/1.0/metadata.xml [hunk 2]
```diff
@@ -76,7 +76,6 @@
   </FILES>
 
   <SYSTEMVARS>
-    <NAME>AUTHORIZED_NETWORKS</NAME>
```
Comment (fanf) [inline techniques/system/common/1.0/metadata.xml:79]: it's still used in common technique as a system var, you need to call it `ALLOWED_NETWORKS` here

**Comment by Fdall** [comment]
Commit modified

**Review by fanf — APPROVED** [review]

**Review by fanf — APPROVED** [review]

**Pull request merged**